src/components/calcite-radio-button/calcite-radio-button.tsx

```diff
@@ -173,8 +179,12 @@ export class CalciteRadioButton {
   }
 
   private uncheckOtherRadioButtonsInGroup(): void {
-    const otherRadioButtons = Array.from(document.querySelectorAll("calcite-radio-button")).filter(
-      (radioButton) => radioButton.name === this.name && radioButton.guid !== this.guid
+    const documentRoot: any = this.el.getRootNode();
```

Comment:
Why not use this.`documentRoot` instead of getting the root node again?

Reply:
Woops, missed that spot,  good catch!